### DISCUSSION TITLE
blog/2023-12-12_mastodon-docker-rootless/

### DISCUSSION THREAD

**Discussion by giscus** [Sieboldianus/du.nkel.dev#16] [Announcements]
# blog/2023-12-12_mastodon-docker-rootless/

A personal code notes blog

https://du.nkel.dev/blog/2023-12-12_mastodon-docker-rootless/

**Comment by Sieboldianus** [comment]
Comments welcome. If you do not want to allow Github API access for [giscus.app](https://giscus.app/), feel free to [directly add comments on Github](https://github.com/Sieboldianus/du.nkel.dev/discussions/16).
